src/Microsoft.TestPlatform.TestHostProvider/Hosting/DotnetTestHostManager.cs

```diff
@@ -506,29 +529,14 @@ public virtual TestProcessStartInfo GetTestHostProcessStartInfo(
         // G:\nuget-package-path\microsoft.testplatform.testhost\version\**\testhost.dll
         // G:\tmp\netcore-test\bin\Debug\netcoreapp1.0\netcore-test.dll
         startInfo.Arguments = args;
-        startInfo.EnvironmentVariables = environmentVariables ?? new Dictionary<string, string?>();
```

Comment:
moved up, otherwise it was overwriting the env variables we set on the start of the method.